.github/workflows/eamxx-gh-ci-standalone.yml

```diff
@@ -0,0 +1,69 @@
+name: gh-standalone
+
+on:
+  pull_request:
+    branches: [ master ]
```

Comment:
Maybe you want to address these later, but I think we may as well polish now before merging...

Can we add also the `types` entry here? I don't want the action to trigger for any pr event. I think `types: [opened, synchronize, ready_for_review, reopened]` is a good set. Otherwise it would trigger for pointless stuff such as adding/removing labels, closing the pr, reassigning...

Reply:
And maybe I would also add a concurrency check. If a commit is pushed while testing, we should kill any other existing PR. Something like

```yaml
concurrency:
  group: ${{ github.workflow }}-${{ github.event_name }}-${{ github.event.pull_request.number || github.run_id }}
  cancel-in-progress: true
```

Btw, I'm not sure why we would want a workflow_dispath trigger here. And if you removed "workflow_dispatch", the group could simply be `${{ github.workflow }}-${{ github.event.pull_request.number}}`. But I don't feel strongly about this, so I'm ok keeping workflow_dispatch.

Reply:
I have to scold my AI bot then. It told me the workflow would trigger for all types of pr event.